apps/tools/blocks-crawler/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "blocks-crawler"
+version = "0.1.0"
+edition = "2021"
+license = "Apache-2.0"
+
+[workspace]
```

Comment:
It should be part of the cargo workspace under apps, do not keep a separate Cargo.lock

Reply:
fixed